Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -179,6 +188,58 @@ public Task<bool> IsContestIpValidByContestAndIp(int contestId, string ip)
                 isUserAdmin);
         }
 
+        public async Task<ContestValidationModel> ValidateContestNew(Contest contest, string userId, bool isUserAdmin,
```

Comment:
This should be inside a `Validation` service. 

Check this code: https://github.com/SoftUni-Internal/interactive/blob/staging/Services/Infrastructure/IValidationServiceAsync.cs

Reply:
Done.